synapse/appservice/api.py

```diff
@@ -326,7 +326,7 @@ async def ping(self, service: "ApplicationService", txn_id: Optional[str]) -> No
         assert service.hs_token is not None
 
         await self.post_json_get_json(
-            uri=f"{service.url}{APP_SERVICE_UNSTABLE_PREFIX}/fi.mau.msc2659/ping",
+            uri=f"{service.url}{APP_SERVICE_PREFIX}/ping",
```

Comment:
I'm assuming we don't really care about supporting backwards compatibility at all?

Reply:
Yeah, the unstable client endpoint and /versions flag were dropped too, so no need for unstable ping to AS

Reply:
Looks like mautrix go & python are the only implementations...which are yours so I'm going to assume you know what you're doing. 😄